example/android/gradle.properties

```diff
@@ -16,3 +16,5 @@
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
+android.enableJetifier=true
```

Comment:
This is not necessary since there are no other dependencies that this library depends on.

Reply:
@se-bastiaan if I remove this line then the sample app fails to build. Maybe because it's running an older version of React Native (0.56.0), which will have Android dependencies that need run through the jetifier?

Reply:
I believe that would be the reason indeed. I'm not sure it is wise to give an example with <0.60 if the update to the package itself will possibly break applications that have not updated to RN 0.60. I'm seeing a lot of different approaches here, I'm not even sure what [the proposed option by the react-native maintainers is exactly](https://github.com/react-native-community/discussions-and-proposals/issues/129#issuecomment-507010053). In the case of this package I think it is a valid approach to drop support for pre-0.60 and go to version 2.0.0, since it heavily depends on the support library/material library to provide the UI. In that case the example would also need to be 0.60.

Reply:
Okay so it seems that a special jetifier tool for react-native is the recommended path to updating: https://github.com/mikehardy/jetifier

Reply:
Agreed, I think we should go to version 2.0.0 👍. 

I have went ahead and updated the example app to React Native 0.60.3. @se-bastiaan, could you give this PR another code review 🙏?

cc @cooperka